dev/bootstrap.sh

```diff
@@ -4,12 +4,14 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-[[ -d "$PWD"/download ]] || mkdir "$PWD"/download
-[[ -d "$PWD"/sstate ]] || mkdir "$PWD"/sstate
+YOCTO_TARGET_ARCH=""
```

Comment:
Maybe let's make the actual value explicit here. Even though it's valid, the fallback to default arch (x86) isn't obvious here.